internal/cli/root.go

```diff
@@ -40,11 +41,13 @@ import (
 const (
 	envPrefix                  = "METAL"
 	configFileWithoutExtension = "metal"
+	DebugVar                   = "PACKNGO_DEBUG"
```

Comment:
This should start with a lower-case letter (`debugVar`)

Reply:
updated